things/api/http/requests.go

```diff
@@ -133,6 +133,22 @@ func (req listMembersReq) validate() error {
 	return nil
 }
 
+type verifyConnectionReq struct {
+	token      string
+	ThingsID   []string `json:"things_id"`
+	ChannelsID []string `json:"channels_id"`
```

Comment:
If we are going to `ThingsID` instead of `thingIDs`, please make it consistant every where in service function to `ThingsID`